osidb/mixins.py

```diff
@@ -532,7 +535,27 @@ def unembargo(self):
 
         # unembargo
         self.set_public()
-
+        refs = pghistory.models.Events.objects.references(self).all()
+        for ref in refs:
+            db, model_name = ref.pgh_model.split(".")
+            model_audit = apps.get_model(db, model_name).objects.filter(
```

Comment:
There's no need to split the name, you can pass it directly
```suggestion
            model_audit = apps.get_model(ref.pgh_model).objects.filter(
```